### PR TITLE
Implement contextual color controls

### PR DESCRIPTION
**Pull request by frankiejarrett**
Implements #61.

It's looking like context doesn't completely replace the need for #56. It may still be necessary.

### CODE REVIEW

**Comment by frankiejarrett** [comment]
@jonathanbardo please review

**Comment by jonathanbardo** [comment]
@fjarrett This is looking great. Nice job on those color scheme man 👍. I say merge it!

File: inc/customizer/colors.php [hunk 1]
```diff
@@ -177,12 +354,31 @@ public function __construct() {
...
+		 * and not filterable.
+		 *
+		 * @since 1.0.0
+		 *
+		 * @var array
+		 */
+		$custom_scheme = array(
+			'_custom' => array(
+				'label' => sprintf( '- %s -' , esc_html__( 'Custom', 'primer' ) ),
+			)
```
Comment (jonathanbardo) [inline inc/customizer/colors.php]: @fjarrett Missing a comma here.

Comment (frankiejarrett) [inline inc/customizer/colors.php]: Good catch!

File: inc/customizer.php [hunk 2]
```diff
@@ -171,7 +171,7 @@ public function use_featured_hero_image( WP_Customize_Manager $wp_customize ) {
 			'use_featured_hero_image',
 			array(
 				'label'       => esc_html__( 'Use featured image', 'primer' ),
-				'description' => esc_html__( "Allow the featured image on the current post to override the header image.", 'primer' ),
+				'description' => esc_html__( "Allow the featured image on the current post to override the hero image.", 'primer' ),
```
Comment (jonathanbardo) [inline inc/customizer.php]: @fjarrett No need for double quote here.

**Pull request merged**